### DISCUSSION TITLE
Is it possible to display a png image when clicking on a feature

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#5307] [Q&A]
Is it possible to display a png image when clicking on a feature in a feature collection layer?

Thanks!

**Comment by hardreddata** [comment]
Check out the ABC Photo Stories layer on https://map.terria.io/ which you get when you clone TerriaMap

**Reply by lstillwe** [reply]
Does this only work for CSV? I tried GeoJSON and that seemed a bit buggy,
